spec/integration/basic_publisher.rb

```diff
@@ -0,0 +1,22 @@
+require "spec_helper"
```

Comment:
I think we need to rename this file to `basic_publisher_spec.rb` so that rspec will pick it up and run it by default